glean-core/metrics.yaml

```diff
@@ -706,6 +706,20 @@ glean.database:
       - glean-team@mozilla.com
     expires: never
 
+  rkv_load_result:
+    type: string
+    description: |
+      The result of attempting to load the RKV database.
```

Comment:
Should we mention here more precisely that it's only the errors and that it is not reported on successful loads?